screens/intro.js

```diff
@@ -90,13 +115,36 @@ export default class Intro extends React.Component {
             This is a collaboration project between de Souza Institute and MGMT Capstone Team 8
           </Text>
         </View>
+        <View style={styles.slide1}>
+          <Image 
+            source={require('../assets/onboarding3.png')}
+            style={{width: 300, height: 300}}
+          />
+          <Text style={{fontWeight: "bold", textAlign: "center", fontSize: 20}}>
+            SleepWell will walk you through an updated version of the Sleeping Well Manual developed by Dr. Sheila Garland 
```

Comment:
Maybe instead we say it is based on the sleep well manual, as opposed to walk you through